src/main/java/org/mifos/connector/common/zeebe/ZeebeClientConfiguration.java

```diff
@@ -1,29 +1,29 @@
-package org.mifos.connector.common.zeebe;
-
-import io.camunda.zeebe.client.ZeebeClient;
-import org.springframework.beans.factory.annotation.Value;
-import org.springframework.context.annotation.Bean;
-import org.springframework.context.annotation.Configuration;
-
-import java.time.Duration;
-
-@Configuration
-public class ZeebeClientConfiguration {
-
-    @Value("${zeebe.broker.contactpoint}")
-    private String zeebeBrokerContactpoint;
-
-    @Value("${zeebe.client.max-execution-threads}")
-    private int zeebeClientMaxThreads;
-
-    @Bean
-    public ZeebeClient setup() {
-        return ZeebeClient.newClientBuilder()
-                .gatewayAddress(zeebeBrokerContactpoint)
-                .usePlaintext()
-                .defaultJobPollInterval(Duration.ofMillis(1))
-                .defaultJobWorkerMaxJobsActive(2000)
-                .numJobWorkerExecutionThreads(zeebeClientMaxThreads)
-                .build();
-    }
-}
+//package org.mifos.connector.common.zeebe;
+//
+//import io.camunda.zeebe.client.ZeebeClient;
+//import org.springframework.beans.factory.annotation.Value;
+//import org.springframework.context.annotation.Bean;
+//import org.springframework.context.annotation.Configuration;
+//
+//import java.time.Duration;
+//
+//@Configuration
+//public class ZeebeClientConfiguration {
+//
+//    @Value("${zeebe.broker.contactpoint}")
```

Comment:
remove commented lines

Reply:
I need to remove it for now because its causing conflict issue with current channel implementation but I do not want to delete the file as it was part of this: https://github.com/openMF/ph-ee-connector-common/pull/20